content/Events/_index.md

```diff
@@ -1,8 +1,20 @@
 ---
 title: Events 
-description: Events in the Kubernetes community
-calendar: true
+description: Events in the Kubernetes Community
 ---
-{{< calendar >}}
 
-See it on the web at [calendar.google.com](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles) , or paste this [iCal url](https://calendar.google.com/calendar/ical/cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com/public/basic.ics) into any [iCal client](https://en.wikipedia.org/wiki/ICalendar). Do NOT copy the meetings over to a your personal calendar, you will miss meeting updates. Instead use your client's calendaring feature to say you are attending the meeting so that any changes made to meetings will be reflected on your personal calendar. 
\ No newline at end of file
+Contributor Summits bring together new and current Kubernetes contributors to connect and learn from one another. It is an opportunity for existing contributors to help shape the future of the project, and offers a welcoming space for new community members to learn, explore and put the contributor workflow to practice.
+
+In some sense, Contributor Summits are a real-life extension of the community and SIG meetings. They aim to:
+
+- Onboard new contributors to be productive in our community
+- Send contributors home with more context, knowledge, and skills about the project
+- Facilitate SIG-specific face-to-face collaboration
+
+Contributor Summits often have three tracks:
+- **New Contributor Workshop** for those looking to find out how to become Kubernetes contributors.
+- **Active Contributor Track** featuring technical discussions, unconference sessions, workshops, and more for those that are actively contributing to the Kubernetes project.
+- The **Documentation Sprint** is for community members who want to help with writing or localizing documentation.
```

Comment:
This should be moved over to the contributor summit page. -- Like this would be better above the upcoming events section.

The landing page for events would probably be better to have a brief description of each sub-section or page under events.